rspace/src/main/scala/coop/rchain/rspace/history/Trie.scala

```diff
@@ -16,9 +16,11 @@ case class NodePointer(hash: Blake2b256Hash) extends NonEmptyPointer
 case class LeafPointer(hash: Blake2b256Hash) extends NonEmptyPointer
 case object EmptyPointer                     extends Pointer
 
-sealed trait Trie[+K, +V]                                          extends Product with Serializable
-final case class Leaf[K, V](key: K, value: V)                      extends Trie[K, V]
-final case class Node(pointerBlock: PointerBlock)                  extends Trie[Nothing, Nothing]
+sealed trait Trie[+K, +V]                     extends Product with Serializable
+final case class Leaf[K, V](key: K, value: V) extends Trie[K, V]
+final case class Node(pointerBlock: PointerBlock) extends Trie[Nothing, Nothing] {
+  def update(item: (Int, Pointer)): Node = Node(pointerBlock.updated(List(item)))
+}
```

Comment:
don't feel strongly about this, but I think it makes the code a bit easier to read.

Reply:
It would make sense to add an overloaded `PointerBlock.updated` method taking a single element instead of a list

Reply:
tell me mere. instead of this? or additionally?